iree/turbine/kernel/_support/tracing.py

```diff
@@ -111,6 +131,12 @@ def proxy(self, node: fx.Node) -> fx.Proxy:
                 node.meta["dtype"] = t
                 node.meta["symbolic_type"] = []
                 self.current_arg_id += 1
+            elif issubclass(t, SymbolBind) and node.op == "placeholder":
+                node.meta["arg_id"] = self.current_arg_id
+                node.meta["symbol_name"] = self.arg_names[self.current_arg_id]
```

Comment:
Why do you need the symbol_name? Isn't the current_arg_id a unique identifier for the argument?